cdap-ui/app/logviewer/main.js

```diff
@@ -0,0 +1,330 @@
+/*
+ * Copyright © 2017 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+console.time(PKG.name);
+
+angular
+  .module(PKG.name, [
+
+    angular.module(PKG.name+'.features', [
+      PKG.name+'.feature.logviewer'
+    ]).name,
+
+    angular.module(PKG.name+'.commons', [
+
+      angular.module(PKG.name+'.services', [
+        'ngAnimate',
+        'ngSanitize',
+        'ngResource',
+        'ngStorage',
+        'ui.router',
+        'cask-angular-window-manager',
+        'cask-angular-theme',
+        'cask-angular-focus',
+        'ngCookies'
+      ]).name,
+
+      angular.module(PKG.name+'.filters', [
+        PKG.name+'.services',
+        'cask-angular-capitalize'
+      ]).name,
+
```

Comment:
We could probably remove half of these. Will drastically reduce the build size though.